Appearances.css

```diff
@@ -0,0 +1,481 @@
+    top: 0px;
+    left: 0;
+    width: 100%;
+    height: 6px;
+    z-index: 99990;
+}
+
+#progress-bar {
+    height: 53%;
+    width: 0;
+    background: linear-gradient(90deg, rgb(255, 0, 0) 0%, rgb(255, 153, 153) 50%, rgb(250, 0, 0) 100%);
+    box-shadow: 0 0 4px rgba(255, 0, 0, 0.7), 0 0 10px rgba(192, 100, 2, 0.7);
+    transition: width 0.09s ease-in-out;
+    border-radius: 10px;
+}
+
+/* Animation */
+.DoAnimation {
+    animation-name: Cinematicuni;
+    animation-duration: 2s;
+    animation-iteration-count: 1;
+    animation-timing-function: ease;
+}
+
+@keyframes Cinematicuni {
+    from {
+        transform: translateX(-450px);
+    }
+    to {
+        transform: translateX(0px);
+    }
+}
+
+/* Navbar */
+
+.navbar{
+    height: 85px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    position: sticky;
+    top: 0;
+    z-index: 1;
+}
+.background{
+  background: rgba(0, 0, 0, 0.8) url(assets/bg3.jpg);
+  background-size: cover;
+  background-blend-mode: darken;
+}
+.background1{
+  background: rgb(0, 0, 0, 0.5) url(assets/bg3.jpg);
+  background-size: cover;
+  background-blend-mode: darken;
+}
+.nav-list{
+  width: 75%;
+  display: flex;
+  align-items: center;
+  margin-right: 75px;
+}
+.nav-list li{
+  list-style: none;
+  padding: 10px 20px;
+  font-size: 18px;
+  margin: 0px 5px;
+}
+.nav-list li a{
+  text-decoration: none;
+  color: white;
+  font-size: 21px;
+  font-family: 'Alkatra', cursive;
+  transition: color 0.3s ease, text-shadow 0.4s ease-in-out;
+}
+.nav-list li a:hover{
+  color: rgb(238, 10, 10);
+  border-radius: 8px;
+  text-shadow: 2px 2px 15px rgba(255, 238, 107, 0.847);
+}
+.nav-list li a.active{
+  color: rgb(238, 10, 10);
+  text-decoration: underline;
+}
+
+/* Right Nav */
+
+ .rightNav{
+  width: 40%;
+  text-align: right;
+  padding: 0px 23px;
+} 
+
+ .rightNav button {
+  color: black;
+  border: none;
+  background-color: rgb(238, 10, 10);
+  margin-bottom: 10px;
+} 
+
+.search-bar{
+  margin-left: 90px;
+  width:300px;
+  height:40px;
+  max-width: 600px;
+  display: flex;
+  align-items: center;
+  background-color: white;
+  border-radius: 60px;
+  padding: 5px 10px;
+}
+.search-bar input{
+  background: transparent;
+  font-family: 'Alkatra', cursive;
+  flex: 1;
+  border: none;
+  outline: none;
+  font-size: 15px;
+  align-items: center;
+  padding-left: 25px;
+  color: rgb(238, 10, 10);
+  margin-top: 3px;
+}
+::placeholder{
+  color: rgb(238, 10, 10);
+}
+
+.search-bar button{
+  border: 0;
+  border-radius: 50%;
+  width: 30px;
+  height: 30px;
+  cursor: pointer;
+  transition: 0.3s;
+  margin-top: 9px;
+  margin-right: 7px;
+}
+.search-bar button:hover{
+  transform: scale(1.1);
+}
+.box-main{
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  color: white;
+  font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+  max-width: 50%;
+  margin: auto;
+  height: 80%;
+}
+
+  
+/* Adjusted Logo */
+.logo img {
```

Comment:
Please see that the main logo of the website is reduced in size.

Original : 
![image](https://github.com/user-attachments/assets/571131d8-e75b-42b3-afa1-1a0f8dd9e734)

After your changes - 
![image](https://github.com/user-attachments/assets/04a4b364-55bb-4ae6-a8f2-a624dc3fd366)